readme.md

```diff
@@ -137,6 +137,8 @@ Redirect Uris that you can use are:
  **https://localhost:45100/signin-oidc**  
  **https://localhost:45100/signin**  
 
```

Comment:
This information is correct that the actual authenticate endpoint. However, as in the example, the oauth framework does this for you. It is out of scope for this example to explain the low-level details of oauth/oidc.